### DISCUSSION TITLE
Definition of Done: Development

### DISCUSSION THREAD

**Discussion by eliasfaltin** [GECO-Project-App/facilitation-app#50] [How we work]
I propose the following definition of done for development work on the project:

- [ ] A reviewer approved Your PR
- [ ] The PR was successfully merged into the main branch
- [ ] You passed all tests
- [ ] Your change was deployed successfully to [app.projectgeco.com](app.projectgeco.com)
- [ ] We successfully used your changes during our internal meetings as the core team 

<details><summary>Comments:</summary>
<p>

- Right now, we only have automated code tests courtesy of GitHub. We should add our own tests later to avoid regressions.

- The purpose of "successfully" using your change in the team is to have a basic QA process. This might be more or less involved depending on the PR in question. For a full exercise or changes to an exercise, we need to "play" the exercise as a team to test how well it performs. In those cases, someone other than me (Elias) should be the facilitator and use the app to run the exercise. 

- For smaller changes, like button updates, etc. we can all, collectively, check them out on our personal devices during a stand up and confirm the tasks for this change is "done".
</p>
</details>

**Comment by alice-ang** [comment]
I think this sounds good! Only comment is if we should add Storybook as a check as well. Depending on what kind of test we're choosing to do, we can create a lot of different tests in Storybook 😄 
https://storybook.js.org/docs/writing-tests

**Comment by eliasfaltin** [comment]
Any text in the app should have the ability to be changed and localized before a feature containing text can be considered done.
